README.md

```diff
@@ -3,6 +3,17 @@
 [![PyPI Status Badge](https://badge.fury.io/py/tensorflow-addons.svg)](https://pypi.org/project/tensorflow-addons/)
 [![Gitter chat](https://img.shields.io/badge/chat-on%20gitter-46bc99.svg)](https://gitter.im/tensorflow/sig-addons)
 
+## Continuous build status
```

Comment:
Looks great! How about removing the line to leave room for introduction below?

Reply:
Hey @facaiy I'm a bit confused by what you mean? Should I just move the introduction to the top after the Gitter chat?